proxy_agent/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure-proxy-agent"
-version = "9.9.9"     # always 3-number version
+version = "1.1.0"      # always 3-number version
```

Comment:
is this version change required ?

Reply:
yes we need to give RSM a version of 1.1 and higher so the can pull a minor version